### ISSUE TITLE
[From: Corey] Apparent version mismatch: Mac - JVM

### ISSUE THREAD

**Issue by capitanevs** [lucarebuffi/WISE#7]
>  I’m running the Oracle Linux VM package of OASYS on a windows machine, and the results I get on my system don’t match Daniele’s on his Mac. When I run the simulation I can’t get the software to simulate the intensity on the mirror properly, and the detector images don’t look right.

**Comment by lucarebuffi** [comment]
Are you running the virtual machine in Oracle VirtualBox? which version? did you installed the guest additions?

**Comment by chardin-slac** [comment]
Yes, I am running the virtual machine in Oracle VirtualBox. I downloaded the preconfigured .vdi, then installed the add-ons within OASYS. I am using WISE version 1.0.22, VirtualBox version 5.1.8.

**Comment by lucarebuffi** [comment]
Thank you, these informations are extremely important to exclude possible sources of error. I will dig into next week, since I am outside Elettra and I cannot access my test environments.
